### PR TITLE
Fix sending PlaybackInfo

### PR DESCRIPTION
**Pull request by crobibero**
Fixes https://github.com/jellyfin/jellyfin/issues/4311

### CODE REVIEW

File: Jellyfin.Api/Controllers/MediaInfoController.cs [hunk 1]
```diff
@@ -111,18 +114,17 @@ public async Task<ActionResult<PlaybackInfoResponse>> GetPlaybackInfo([FromRoute
...
+            [FromQuery] bool? enableTranscoding,
+            [FromQuery] bool? allowVideoStreamCopy,
+            [FromQuery] bool? allowAudioStreamCopy,
+            [FromBody] PlaybackInfoDto? playbackInfoDto)
         {
             var authInfo = _authContext.GetAuthorizationInfo(Request);
 
-            var profile = deviceProfile?.DeviceProfile;
-
+            var profile = playbackInfoDto?.DeviceProfile?.DeviceProfile;
```
Comment (oddstr13) [inline Jellyfin.Api/Controllers/MediaInfoController.cs:127]: That doesn't look right…
Does this move the device profile one level deeper in the JSON payload?

**Review by nielsvanvelzen — COMMENTED** [review]
Is it possible to create two separate operations for this?

File: Jellyfin.Api/Controllers/MediaInfoController.cs [hunk 2]
```diff
@@ -183,9 +202,9 @@ public async Task<ActionResult<PlaybackInfoResponse>> GetPlaybackInfo([FromRoute
                         new LiveStreamRequest
                         {
                             AudioStreamIndex = audioStreamIndex,
-                            DeviceProfile = deviceProfile?.DeviceProfile,
-                            EnableDirectPlay = enableDirectPlay,
-                            EnableDirectStream = enableDirectStream,
+                            DeviceProfile = playbackInfoDto?.DeviceProfile?.DeviceProfile,
```
Comment (oddstr13) [inline Jellyfin.Api/Controllers/MediaInfoController.cs:205]: :point_up:

Comment (crobibero) [reply Jellyfin.Api/Controllers/MediaInfoController.cs:205]: It does, but only because it was migrated incorrectly

**Review by oddstr13 — APPROVED** [review]

**Review by BaronGreenback — APPROVED** [review]

**Pull request merged**